moccasin/config.py

```diff
@@ -9,16 +9,15 @@
 
 import boa
 import tomlkit
-
-# If we want to know the specific deployer that the active env is working with
-# from boa.interpret import _get_default_deployer_class
 from boa.contracts.abi.abi_contract import ABIContract, ABIContractFactory
 from boa.contracts.vyper.vyper_contract import VyperContract, VyperDeployer
 from boa.deployments import Deployment, get_deployments_db
 from boa.environment import Env
 from boa.util.abi import Address
 from boa.verifiers import get_verification_bundle
-from boa_zksync import set_zksync_fork, set_zksync_test_env
+from boa_zksync import set_zksync_env, set_zksync_fork, set_zksync_test_env
+
+# REVIEW: Could/should these be optionally imported?
```

Comment:
it looks like it's only used for typing, so it could be in TYPE_CHECKING.
Still, that would require boa_zksync to be installed.

Ideally all contracts should work the same way so we can just use `BaseEvmContract`